.github/workflows/ci.yml

```diff
@@ -0,0 +1,129 @@
+name: windows_exporter CI/CD
+  
+# Trigger on pull requests and releases
+# Deployments will only occur for releases (see `if` clauses in the build job).
+on:
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+      - published
+      - edited
+
+jobs:
+  test:
+    runs-on: windows-2019
+    env:
+      GO111MODULE: on
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Test
+        run: make test
+
+      - name: Install e2e deps
+        run: |
+          go get -u github.com/prometheus/promu@v0.11.1
+          go get -u github.com/josephspurrier/goversioninfo/cmd/goversioninfo@v1.2.0
+          # GOPATH\bin dir must be appended to PATH else the `promu` command won't be found
+          echo "$(go env GOPATH)\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+
+      - name: e2e Test
+        run: make e2e-test
+
+  lint:
+    runs-on: windows-2019
+    env:
+      GO111MODULE: on
+    steps:
+      # `gofmt` linter run by golangci-lint fails on CRLF line endings (the default for Windows)
+      - name: Set git to use LF
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.eol lf
+
+      - uses: actions/checkout@v2
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.43
+          args: "--timeout=5m"
+
+      # golangci-lint action doesn't always provide helpful output, so re-run without the action for
+      # better output of the problem.
+      # The cache from the golangci-lint step is re-used here, so this step should finish quickly.
+      - name: errors
+        if: ${{ failure() }}
+        run: golangci-lint run --timeout=5m -c .golangci.yaml
+
+  build:
+    runs-on: windows-2019
+    needs:
+      - test
+      - lint
+    env:
+      GO111MODULE: on
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # fetch-depth required for gitversion in `Build` step
+          fetch-depth: 0
+
+      - name: Install Build deps
+        run: |
+          go get -u github.com/prometheus/promu@v0.11.1
+          go get -u github.com/josephspurrier/goversioninfo/cmd/goversioninfo@v1.2.0
+          # GOPATH\bin dir must be added to PATH else the `promu` and `goversioninfo` commands won't be found
+          echo "$(go env GOPATH)\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+
+      - name: Build
+        run: |
+          # go mod download (or, if we don't call it, go build) will write every dependent package name to
+          # stderr, which will be interpreted as an error and abort the build if ErrorActionPreference is Stop,
+          # so we need to run it before setting the preference.
```

Comment:
Just checking - is this the case also for Github Actions?

Reply:
I've just tested and no, this must have been an Appveyor quirk. I've fixed this in a recent push.